### ISSUE TITLE
Review buttons use all-caps

### ISSUE THREAD

**Issue by amire80** [commons-app/apps-android-commons#2873]
Buttons such as "NO, MIS-CATEGORIZED", "SEEMS FINE", "NO, OUT OF SCOPE", etc., use all-caps. This is a bad practice for translatable software user interface strings, because most alphabets don't have capital letters, and in alphabets that do have them this has questionable readability.

All of them should be changed to regular capitalization.

**Comment by neslihanturan** [comment]
Thanks for letting us know @amire80 . They are capital because the design guideline we followed suggests using capital for such buttons when the language support capital letters: https://material.io/design/components/buttons.html#anatomy 
Would it be helpful for translations if we use non-all-capital strings in string resources and capitalize them programmatically for buttons only if the language let us to do?

**Comment by amire80** [comment]
Thanks for the link.

It's good to see that the Material design guidelines acknowledge that only some languages have capitalization, and explains that the purpose is "to distinguish the text label from surrounding text". However, it's unfortunate that it doesn't provide any solution for distinguishing the text label in languages that don't have capitalization.

The same guidelines also say: "Don’t wrap text. For maximum legibility, a text label should remain on a single line." Since some of the labels that we are discussing are quite long, I suspect that it won't be possible to follow this guideline with them.

Perhaps this screen should not use buttons at all, but something that is more like a menu. This will allow readable long labels and remove any need for capitalization.

**Comment by neslihanturan** [comment]
> "to distinguish the text label from surrounding text". However, it's unfortunate that it doesn't provide any solution for distinguishing the text label in languages that don't have capitalization.

Styling, coloring can be options

> The same guidelines also say: "Don’t wrap text. For maximum legibility, a text label should remain on a single line." Since some of the labels that we are discussing are quite long, I suspect that it won't be possible to follow this guideline with them.

I agree we should have use shorter strings. 

 This was a new feature implemented during the previous hackathon, thus its UI planned in very short time. Let's wait opinions from others, maybe we can consider complete new UI as you suggested.

**Comment by neslihanturan** [comment]
After #2916 is merged, we can reconsider Review Activity buttons. 
- Currently @ashishkumar468 fixed all capital strings problem from strings.xml, so that translators should see normal texts.
- We use textAllCaps=true for capitalizing strings for later use in buttons. According to documentation https://developer.android.com/reference/android/widget/TextView#attr_android:textAllCaps this method uses capital letters when "available". So we should be good.

I think everything about this issue seems solved, so closing it. Please reopen for further discussions @amire80

**Comment by sherlockbeard** [comment]
#2999  in this pull i made the necessary changes required in layout files and in the string value file
